go.sum

```diff
@@ -1,93 +0,0 @@
-github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```

Comment:
do we need these files after all or no?

Reply:
we needed them in the previous version, we changed the way caddy is installed, hence the removal